### DISCUSSION TITLE
Svelte 5 support

### DISCUSSION THREAD

**Discussion by adriablancafort** [pevey/sveltekit-medusa-client#6] [Q&A]
Hello Pevey,

Are there any plans to add support for svelte 5?
When installing it in a svelte project it fails due to:

```
npm error Could not resolve dependency:
npm error peer svelte@"^4.0.0"
```

**Comment by pevey** [comment]
Just noting for others that the OP and I discussed this a bit on Discord.  Summary:

Now that:
- Medusa 2.0 is out
- The medusajs/js-sdk package is much leaner than it used to be
- Svelte 5 is officially released

I'm going to be looking at the sveltekit-medusa-starter to update it.  Probably (though not yet confirmed), this package will not be necessary.
